package.json

```diff
@@ -1,8 +1,31 @@
 {
+  "name": "this-week-in-react",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "react-static start",
+    "stage": "react-static build --staging",
+    "build": "react-static build",
+    "serve": "serve dist -p 3000"
+  },
+  "dependencies": {
+    "chokidar": "2.0.4",
+    "htmr": "0.7.0",
+    "jdown": "^0.6.5",
```

Comment:
The same goes for this dependency.

Reply:
I was using it to convert a directory of markdown to html but the repo does not need it now. I will remove it.